front/app/components/Form/Components/Controls/MapInput/multiPointUtils.tsx

```diff
@@ -369,7 +369,12 @@ export const updateMultiPointsDataAndDisplay = ({
       const graphicsLayer = new GraphicsLayer({ title: 'User Input' });
       graphicsLayer.add(lineGraphic);
       graphicsLayer.addMany(pointGraphics);
-      mapView.map.add(graphicsLayer);
+
+      // mapView.map is not always defined, so we do need to check here.
+      // The type comes from ArcGIS, so we can't change it.
+      // We need to include the eslint comment below.
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+      mapView.map?.add(graphicsLayer);
     }
```

Comment:
Seems like the type was also wrong from ArcGIS here - sometimes we do have a mapView with no map object yet :thinking:

Reply:
@amanda-anderson Let's still emerge this part?

Reply:
Yes I'll just make a separate PR :)